root/static/scripts/common/components/TagEditor.js

```diff
@@ -512,18 +513,20 @@ export const MainTagEditor = hydrate<TagEditorProps>(
 
           {(positiveTagsOnly && !tags.every(isAlwaysVisible)) ? (
             <>
-              {this.props.$c.user ? (
-                <p>
-                  {l(
-                    `Tags with a score of zero or below,
-                     and tags that you’ve downvoted are hidden.`,
-                  )}
-                </p>
-              ) : (
-                <p>
-                  {l('Tags with a score of zero or below are hidden.') + ' '}
-                </p>
-              )}
+              {this.props.$c.user &&
+                this.props.$c.user.has_confirmed_email_address ? (
+                  <p>
+                    {l(
+                      `Tags with a score of zero or below,
+                      and tags that you’ve downvoted are hidden.`,
+                    )}
+                  </p>
+                ) : (
+                  <p>
+                    {l('Tags with a score of zero or below are hidden.')}
+                    {' '}
```

Comment:
This change seems to be completely unrelated, plus this extra space seems to be useless anyway.

Reply:
IIRC it made the line too long otherwise? We were already adding the extra space, I just moved it.

Reply:
It does seem to be useless though, yes. Just removed it.